text/0000-qa-team.md

```diff
@@ -0,0 +1,106 @@
+* Any necessary feedback is shared with the contributor to empower them to continue to make high quality contributions
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+## Membership
+
+* Membership is bootstrapped by @lehnberg and @j01tz
+* New membership approved with consensus of existing team members
+* Members may be removed by self-resignation, consensus decision by QA team members or by core team
+
+## Decision Making
+
+* A consensus-seeking approach is used for decision making through dialogue and discussion, modeled after the general principle in the Grin governance RFC[1].
+* In the event the team cannot reach consensus on a required action, the core team will determine the outcome
+
+## Responsibilities
+
+The responsibilities of the QA team _do_ include:
+  * Establishing and providing the necessary support for contributors to have a clear path to success by empowering them to perform quality assurance testing and review on their own work
+  * Clarifying, monitoring and enforcing the following of best QA practices by contributors
+
+The responsibilities of the QA team _do not_ include:
+  * Doing the quality assurance testing and review work themselves
+  * Writing tests for code submissions and changes
+  * Keeping testing frameworks updated based on codebase changes
+  * Manually reviewing each submission for quality assurance
+  * _Was the codebase changed? Then the tests must also be changed or extended for full coverage- this is not the responsibility of the QA team!_
+
+## Community Contributions
+
+The QA team is ultimately responsible for creating an environment that encourages and empowers contributors to make quality contributions to Grin. To this end, the QA team will strive to provide the resources necessary for contributors to make consistent, high quality contributions to Grin, including but not limited to: documentation, constructive feedback, tooling and clarity of expectations of quality.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+* Resources are already spread thin, don't want to assign responsibilities if people won't have time
+
+# Rationale and alternatives
+[rationale-and-alternatives]: #rationale-and-alternatives
+
+The alternative is a volunteer based approach to QA. This often leaves areas not properly assessed for quality that volunteers do not find interesting are already too busy making or reviewing other contributions. A volunteer based QA approach does not establish any accountability or oversight to changes in quality over time, nor does it provide active efforts to support new contributors.
+
+# Prior art
+[prior-art]: #prior-art
+
+The QA team RFC is following the guidance of the establishment of teams in the governance RFC[0].
+
+# Unresolved questions
+[unresolved-questions]: #unresolved-questions
+
+* To what extent does the QA team develop tools?
```

Comment:
This part of it is key to me. In my mind an effective QA takes a very active role in the creation of tools, frameworks and environments that continually deploy and test changes. Of course this can be supported by developers, but the QA team very much needs to drive the overall testing vision and take the leadership to ensure the correct tools are in place.

Reply:
I think it would be reasonable to expect the QA team to develop and make available the tools for sufficient testing. This looks like a big initial effort to get a test harness to support the node, wallet and miner. Additionally if new aspects emerge that require integration, an initial effort may be required to glue the necessary pieces so that contributors can begin adding and testing those as well. Beyond those initial efforts to add support to unsupported areas, as currently written, any changes should really be the responsibility of contributors. 

However if the QA team doesn't give contributors everything possible to succeed (good documentation, sane frameworks, stationary goalposts etc.) then the responsibility comes back to the QA team in my opinion. There's probably room for a lot of discussion and clarification here. For an ideal QA team, technical vision and leadership is essential. Whether this is necessary for a minimally viable QA team seems open (as is the question of whether a minimally viable QA team is worth it).